tests/foreman/cli/test_contentviews.py

```diff
@@ -1088,34 +1000,33 @@ def test_cv_promote_composite(self):
             u'id': con_view['id'],
             u'content-view-version-id': version1_id,
         })
-        self.assertEqual(result.return_code, 0,
-                         "Repo was not associated to selected CV")
-        self.assertEqual(len(result.stderr), 0, "No error was expected")
+        self.assertEqual(
+            result.return_code, 0,
+            "Repo was not associated to selected CV"
+        )
```

Comment:
I have reviewed some of the messages, will do it again. There are many files and tests to look over.